electron-app/src/views/settings/GeneralSettings.vue

```diff
@@ -0,0 +1,543 @@
+<template>
+  <v-container id="settings-general">
```

Comment:
These are leftovers from porting from the jQuery version but could you take care of that too please? :D
again class `general-settings` and the same pattern though the classes?

Reply:
Sure.